orders.go

```diff
@@ -82,6 +82,8 @@ type Order struct {
 	SubtotalPrice string
 	// SubtotalPriceSet is the subtotal of the order in shop and presentment currencies after discounts but before shipping, duties, taxes, and tips.
 	SubtotalPriceSet PriceSet
+	// Tags attached to the order. Tags are additional short descriptors, commonly used for filtering and searching. Each individual tag is limited to 40 characters in length.
+	Tags []string
```

Comment:
Just to double check, should this not be a string? I thought it just mirrored what's in the [api docs](https://shopify.dev/api/admin/rest/reference/orders/order)?

Reply:
Technically yes, I've got the DTO as just a string in httpshopify (I've tested that it works), but I think this makes working with it much easier. It's something we can have a genuine discussion about.

Reply:
Actually discussed it earlier with Harry. I think like @connor-mohc said... this should map Shopify really. That's what we are always trying to do BUT there's nothing stopping us from having a new type in this package that can have a function that basically splits it into a `[]string` that makes it easier to use for our use.

Not sure what you think of that guys.

Reply:
Hmm :thinking: Might be best for @harry-mohc or @Ianmohc to answer that one. My understanding is this package should just mirror the Shopify API. I guess with this change, we open ourselves up to slightly modifying other properties and then we slowly drift away to what is in Shopify? 

Also slightly odd that in orders `Tags` is a `[]string` but in products it's a `string`.

Reply:
`type CommaSeperated string`

```golang
func (string CommaSeperated) toSlice() []string {
    return strings.Split(string, ",")
}
```
????